### ISSUE TITLE
[ BUG ] `Import-FalconConfig` fails to create `SensorUpdatePolicy` items.

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#301]
**Describe the bug**
`Import-FalconConfig` fails to create `SensorUpdatePolicy` items.

**To Reproduce**
Create a configuration export containing `SensorUpdatePolicy` and use `Import-FalconConfig`, 

**Expected behavior**
Successful creation of `SensorUpdatePolicy` items.

**Environment (please complete the following information):**
 - OS: Windows 11
 - PowerShell: 5.1
 - PSFalcon: 2.2.4

**Comment by bk-cs** [comment]
This is due to the introduction of update scheduling exclusions. `Get-FalconSensorUpdatePolicy` returns a blank `scheduler` value under `settings`, which is invalid for use with `New-FalconSensorUpdatePolicy`. `New-FalconSensorUpdatePolicy` needs to be modified to filter these blank values out to ensure successful policy creation.

**Comment by bk-cs** [comment]
This will be fixed in the v2.2.5 release. You can fix your local module by replacing `Public\policy-sensor-update.ps1`  with the 2.2.5 version:

```powershell
(Invoke-WebRequest https://raw.githubusercontent.com/CrowdStrike/psfalcon/2.2.5/public/policy-sensor-update.ps1).Content > $HOME\Documents\WindowsPowerShell\Modules\PSFalcon\2.2.4\Public\policy-sensor-update.ps1
```
